tests/test_ssm.py

```diff
@@ -47,3 +47,60 @@ def test_ssm_parameter_not_secure(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
         self.addCleanup(client.delete_parameters, Names=['test-name', 'secure-test-name'])
+
+    @functional
+    def test_ssm_activation_expired(self):
+        session_factory = self.replay_flight_data("test_ssm_activation_expired")
+        client = session_factory().client("ssm")
+        p = self.load_policy(
+            {
+                "name": "ssm-list-expired-activations",
+                "resource": "ssm-activation",
+                "filters": [{
+                  "type": "value",
+                  "key": "Expired",
+                  "value": True
+                }]
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 2)
+
+    @functional
+    def test_ssm_activation_expired_no_registrations(self):
```

Comment:
ditto re functional, but more to the point this test isn't adding any value additional to the previous test_ssm_activate_expired and should be removed.